src/server/game/World/WorldState.h

```diff
@@ -0,0 +1,94 @@
+/*
+ * This file is part of the AzerothCore Project. See AUTHORS file for Copyright information
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by the
+ * Free Software Foundation; either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef WORLD_STATE_H
+#define WORLD_STATE_H
+
+#include "Player.h"
+#include <atomic>
+
+enum Conditions
```

Comment:
I feel like the names for the enums here are too generic. Maybe it's due to my lack of C++ knowledge, but I think they might conflict with other elements. If used as a type, adding "world state" context to the names could improve readability.

Reply:
I prefixed with `WORLD_STATE` where needed. It should it should now better match rest of the codebase